applications/StructuralMechanicsApplication/tests/adjoint_sensitivity_analysis_test_factory.py

```diff
@@ -129,13 +129,9 @@ class TestAdjointSensitivityAnalysisBeamStructureNodalReaction(AdjointSensitivit
 
     def perform_additional_checks(self):
         adjoint_model_part = self.adjoint_analysis.model.GetModelPart(self.model_part_name)
-        reference_values = [-0.31249774999397384, -1.249]
-        sensitivities_to_check = []
-        sensitivities_to_check.append(adjoint_model_part.Conditions[1].GetValue(StructuralMechanicsApplication.POINT_LOAD_SENSITIVITY)[2])
-        sensitivities_to_check.append(adjoint_model_part.Elements[10].GetValue(StructuralMechanicsApplication.I22_SENSITIVITY))
```

Comment:
I erased the check of I22_SENSITIVITY because it is already checked by `from_json_check_result_process`

Reply:
Ok